packages/website/src/building-blocs/PageLayout.tsx

```diff
@@ -97,7 +97,7 @@ const Content: FunctionComponent<Pick<
 >> = ({code, examples, id, relatedComponents, layout, withPropsTable, children}) => (
     <>
         <div className="plasma-page-layout__main-code plasma-page-layout__section">
-            <Sandbox id="main-code" horizontal={layout === 'horizontal'}>
+            <Sandbox key={id + 'main-code'} id="main-code" horizontal={layout === 'horizontal'}>
```

Comment:
Fix the single select bug? 👀

Reply:
I was an attempt at it from before innovate, but it doesn't change anything

Reply:
I guess in the end the id does not change for the same page so 🙈  pouette pouette 😆